### PR TITLE
Fix titles link

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes to the docs site code

#### Description

- What does this PR change? Give us a brief description.
Fixed the icon linking the page title to the URL.

* Previous behavior:
image
* Now behavior: 
image

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Comment by Yan-Thomas** [comment]
I can see removing this CSS rule causes the viewport to work less in smaller viewports (I don't think it's a big problem considering most people aren't going to see the site in a 230px wide screen), but here's what's happening:

| Before | After|
|--------|--------|
|image| image |

**Comment by Waxer59** [comment]
I have already added a mediaquery that solves this problem.

**Review by Yan-Thomas — APPROVED** [review]
Thanks, perfect!

**Review by sarah11918 — APPROVED** [review]
I am happy with the discussion I've seen here! This seems to have been a well-thought-out improvement, and I'm happy to approve subject to Yan or Chris being satisfied!  :purple_heart:

**Review by delucis — CHANGES_REQUESTED** [review]
In general we always use `min-width` media queries instead of `max-width`. Can we change this to keep `overflow-x: hidden` the default style and then disable it at `min-width: 50em` instead, please?

**Comment by Waxer59** [comment]
Changes made!

File: src/components/PageContent/PageContent.astro [hunk 1]
```diff
@@ -91,7 +97,7 @@ const direction = rtlLanguages.has(lang) ? 'rtl' : 'ltr';
 			height: 0;
 		}
 	}
-
+
```
Comment (Waxer59) [inline src/components/PageContent/PageContent.astro:100]:
```suggestion
```

**Review by delucis — APPROVED** [review]
Thanks @Waxer59! Made a tweak to avoid using `!important` which is generally best avoid if it’s at all possible. Here we just needed to move the media query `.content` style below the `.content` without a media query. I took the opportunity to group up the different media queries too.

**Review by Yan-Thomas — APPROVED** [review]
LGTM!

**Pull request merged**